MetaMask/metamask-extension
# Fix-BannerAlert to TS

## Explanation

- fixes #19118 

## Screenshots/Screencaps


### Before

https://github.com/MetaMask/metamask-extension/assets/79097544/e74edffd-b1a6-4226-b9eb-0f788ec5d487

### After

https://github.com/MetaMask/metamask-extension/assets/79097544/47397458-fdfc-4647-8b61-167d05fac811

## Manual Testing Steps

- Check storybook controls for `BannerAlert`
- Ensure unit tests pass by running `yarn jest ui/components/component-library/banner-alert`

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
